dir.targets

```diff
@@ -62,6 +62,15 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="GenerateRuntimeConfig" BeforeTargets="Build" Condition="$(OutputType) == 'Exe' ">
+    <ItemGroup>
+      <RuntimeConfigTemplate Include="$(ToolsDir)\xunit.console.netcore.runtimeconfig.json"/>
```

Comment:
I actually meant we could just check in a separate "RuntimeConfigTemplate" file, not necessarily just check in the runtimeconfig for RemoteExecutorConsoleApp. I was just referring to how this stuff here relies on "xunit.console.netcore.runtimeconfig.json" existing, but that could end going away for unrelated reasons.